pulp_rpm/app/upload.py

```diff
@@ -33,6 +34,9 @@ def one_shot_upload(artifact, repository=None):
         relative_path=filename
     )
 
+    resource = CreatedResource(content_object=pkg)
```

Comment:
Since the code above uses "get_or_create()", should we only have a CreatedResource when one was actually created?

Reply:
There is check on line 28 and if not created an exception is raised (so in my understanding it won't reach this line)